operator/v1/types_dns.go

```diff
@@ -287,6 +287,23 @@ type ForwardPlugin struct {
 	//
 	// +optional
 	TransportConfig DNSTransportConfig `json:"transportConfig,omitempty"`
+
+
+	// protocolStrategy specifies the protocol to use for upstream DNS
+	// requests. Valid values for protocolStrategy are "TCP" or empty.
+	// If empty, the default is used, which is currently the following: use
```

Comment:
Would you please rephrase
```
If empty, the default is used, which is currently the following: use
the protocol of the original client request, but this is subject to
change over time.
```
? The sentence `use the protocol of the original client request, but this is subject to change over time` is unclear to me. Whether the whole sentence is the default value. Or, whether the default value can change over time. In a stable API once a default value is set, it can not be changed. Does the default value in question here (the default value that can change over time) depend on specific CoreDNS implementation maybe? Which can change over time?